pkg/framework/deployment.go

```diff
@@ -43,6 +44,17 @@ func DeleteDeployment(c client.Client, deployment *kappsapi.Deployment) error {
 	})
 }
 
+// UpdateDeployment updates the specified deployment
+func UpdateDeployment(c client.Client, deployment *kappsapi.Deployment, updated *kappsapi.Deployment) error {
+	return wait.PollImmediate(1*time.Second, WaitMedium, func() (bool, error) {
```

Comment:
Perhaps we should add a constant for `retryShort` in framework.go? We only have `retryMedium` right now which is 5 seconds